src/coreclr/nativeaot/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -1308,6 +1312,36 @@ REDHAWK_PALEXPORT void REDHAWK_PALAPI PAL_GetCpuCapabilityFlags(int* flags)
 #endif // AT_HWCAP2
 
 #else // !HAVE_AUXV_HWCAP_H
+
+#if HAVE_SYSCTLBYNAME
```

Comment:
This will be needed for Apple M1 eventually. Adding it proactively (copy from CoreCLR PAL).